airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/ConnectionOptions.yaml

```
@@ -0,0 +1,17 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/ConnectionOptions.yaml
+title: ConnectionOptions
+description: List of sandbox cloud connections that tests can be run against
+type: object
+required:
+  - connectionName
+  - connectionId
+additionalProperties: false
+properties:
+  connectionName:
+    type: string
+    description: "The connection name"
+  connectionId:
+    type: string
+    description: "The connection ID"
```

```suggestion
  name:
    type: string
    description: "The connection name"
  id:
    type: string
    description: "The connection ID"
```

As these field are already under a `connection` parent it might be redundant to use a `connection` prefix.